### PR TITLE
Adds test for feed-advanced-search component to increase coverage

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
**Changes proposed in this pull request**
- Added test for feed-advanced-search component to increase coverage

**Link to live demo: ** 
http://pr-944-fossasia-loklaksearch.surge.sh

**Related to #414**

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @simsausaurabh please review :)

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts [hunk 1]
```diff
@@ -30,7 +30,156 @@ describe('Component: FeedAdvancedSearchComponent', () => {
...
+	it('should set toolList on ngOnInit() and blank query', async(() => {
+		const fixture = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const component = fixture.debugElement.componentInstance;
+		component.ngOnInit();
+		expect(component.toolList.inert).toBe(true);
+		const query$ = component.store.select(fromRoot.getQuery);
+		let displayString: string;
+		const subscription = query$.subscribe(query => displayString = query.displayString);
+		expect(displayString).toBe('');
+		subscription.unsubscribe();
```
Comment (simsausaurabh) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts:54]: Is it explicitly required?

**Review by simsausaurabh — COMMENTED** [review]
see inline

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts:54]: @simsausaurabh  Actually in this block of code, I am just testing that `getDataFromStore()` of `feed-advanced-search.component.ts` is returning empty string to `query$` initially.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh @praveenojha33 please review

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts:54]: @simsausaurabh please review this

**Review by simsausaurabh — APPROVED** [review]

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts [hunk 2]
```diff
@@ -30,7 +30,156 @@ describe('Component: FeedAdvancedSearchComponent', () => {
...
+		const component = fixture.debugElement.componentInstance;
+		component.getLocationBasedResults('UK');
+
+		expect(component.locationValue).toBe('Country: UK');
+	}));
+
+	it('should call getLocationBasedResults(else)', async(() => {
+		const fixture = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const component = fixture.debugElement.componentInstance;
+		component.getLocationBasedResults('else');
```
Comment (Achint08) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts:181]: Can you explain the logic behind `component.getLocationBasedResults('else'); ` ? What is the scenario that you are testing?

Comment (shreyanshdwivedi) [reply src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts:181]: @Achint08 here I'm testing else block of `getLocationBasedResults()`

**Review by Achint08 — APPROVED** [review]

**Pull request merged**